### DISCUSSION TITLE
MQTT content configuration

### DISCUSSION THREAD

**Discussion by WirthmU** [aaronwmorris/indi-allsky#1241] [General]
Where are the parameters defined that are in included in the MQTT message?  I would like to remove the current picture from the list.

**Comment by aaronwmorris** [comment]
Currently, there are no options to configure what data is published or not.  I can add a new parameter to disable image publishing with MQTT.

**Reply by WirthmU** [reply]
This would be brilliant. The load with the picture is quit heavy.

**Comment by aaronwmorris** [comment]
Merged #1243 with the image publish option for MQTT.  Let me know how that works.

**Comment by WirthmU** [comment]
It works - this is perfect.
